### PR TITLE
 Change over to Foreman shared GH actions

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@ofedoren can this one get a review too?

**Review by ofedoren — APPROVED** [review]
Well, since the plugin doesn't have any tests, this can be changed to run rubocop only. I guess ACK from me, but I'll leave the final decision to the current maintainer :)

**Pull request merged**
